### PR TITLE
[NEW APP] Lobe Chat

### PR DESCRIPTION
**Pull request by asermax**
Still in early development but plenty popular and pretty powerful GPT interface.

Upstream repo: https://github.com/lobehub/lobe-chat

## Summary by CodeRabbit

- **Documentation**
	- Updated documentation for Lobe Chat, outlining features and configuration instructions.

### CODE REVIEW

File: apps/lobe-chat/config.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  "website": "https://github.com/lobehub/lobe-chat",
+  "form_fields": [
+    {
+        "type": "text",
+        "label": "OpenAI API Key",
+        "required": true,
+        "env_variable": "OPENAI_API_KEY"
+    },
+    {
+        "type": "fqdn",
```
Comment (meienberger) [inline apps/lobe-chat/config.json:23]: This should be an URL
```suggestion
        "type": "url",
```

File: apps/lobe-chat/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    environment:
+      - OPENAI_API_KEY=${OPENAI_API_KEY}
+      - OPENAI_PROXY_URL=${OPEANAI_PROXY_URL}
+      - CUSTOM_MODELS=${CUSTOM_MODELS}
+      - ACCESS_CODE=${ACCESS_CODE}
+    ports:
+      - ${APP_PORT}:3210
+    restart: unless-stopped
+    networks:
+      - tipi_main_network
```
Comment (meienberger) [inline apps/lobe-chat/docker-compose.yml:15]: You are missing the traefik labels to make the app exposable

```suggestion
      - tipi_main_network
    labels:
      # Main
      traefik.enable: true
      traefik.http.middlewares.lobe-chat-web-redirect.redirectscheme.scheme: https
      traefik.http.services.lobe-chat.loadbalancer.server.port: 16662
      # Web
      traefik.http.routers.lobe-chat-insecure.rule: Host(`${APP_DOMAIN}`)
      traefik.http.routers.lobe-chat-insecure.entrypoints: web
      traefik.http.routers.lobe-chat-insecure.service: lobe-chat
      traefik.http.routers.lobe-chat-insecure.middlewares: lobe-chat-web-redirect
      # Websecure
      traefik.http.routers.lobe-chat.rule: Host(`${APP_DOMAIN}`)
      traefik.http.routers.lobe-chat.entrypoints: websecure
      traefik.http.routers.lobe-chat.service: lobe-chat
      traefik.http.routers.lobe-chat.tls.certresolver: myresolver
      # Local domain
      traefik.http.routers.lobe-chat-local-insecure.rule: Host(`lobe-chat.${LOCAL_DOMAIN}`)
      traefik.http.routers.lobe-chat-local-insecure.entrypoints: web
      traefik.http.routers.lobe-chat-local-insecure.service: lobe-chat
      traefik.http.routers.lobe-chat-local-insecure.middlewares: lobe-chat-web-redirect
      # Local domain secure
      traefik.http.routers.lobe-chat-local.rule: Host(`lobe-chat.${LOCAL_DOMAIN}`)
      traefik.http.routers.lobe-chat-local.entrypoints: websecure
      traefik.http.routers.lobe-chat-local.service: lobe-chat
      traefik.http.routers.lobe-chat-local.tls: true
```

**Review by meienberger — CHANGES_REQUESTED** [review]
Thank you @asermax, the app works fine. Here are a few requests before we can merge it

**Comment by asermax** [comment]
For sure, will make those changes, thanks for the review!

**Comment by meienberger** [comment]
Thanks for the changes @asermax

**Pull request merged**